src/main/java/com/org/ita/kata/implementation/MyroslavaVynokur/SevenImpl.java

```
@@ -4,8 +4,17 @@
 
 public class SevenImpl implements Seven {
     @Override
-    public long newAvg(double[] arr, double navg) {
-        return 0;
+    public long newAvg(double[] arr, double navg) throws IllegalArgumentException {
```

need to throw the error above?

---

removed error